mongodbatlas/transition_test.go

```diff
@@ -0,0 +1,119 @@
+package mongodbatlas_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/config"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+)
+
+type MongoDBClient = config.MongoDBClient
```

Comment:
same case here

Reply:
once we agree on file structure, I'd prefer to do it in a different PR that ca be done just after so it's easier to track changes and we avoid to increase this PR more